core/src/main/res/values/strings.xml

```diff
@@ -403,104 +403,104 @@
     <string name="pref_expandNotify_title">High Notification priority</string>
-    <string name="pref_expandNotify_sum">This usually expands the notification to show playback buttons.</string>
+    <string name="pref_expandNotify_sum">Usually expands the notification to show playback buttons.</string>
     <string name="pref_persistNotify_title">Persistent Playback Controls</string>
     <string name="pref_persistNotify_sum">Keep notification and lockscreen controls when playback is paused.</string>
-    <string name="pref_compact_notification_buttons_title">Set Compact Notification Buttons</string>
-    <string name="pref_compact_notification_buttons_sum">Change the playback buttons when the notification is collapsed. The play/pause button is always included.</string>
-    <string name="pref_compact_notification_buttons_dialog_title">Select a maximum of %1$d items</string>
-    <string name="pref_compact_notification_buttons_dialog_error">You can only select a maximum of %1$d items.</string>
+    <string name="pref_compact_notification_buttons_title">Compact Notification Buttons</string>
+    <string name="pref_compact_notification_buttons_sum">Change the playback buttons when the notification is collapsed. (Play/pause is always shown.)</string>
+    <string name="pref_compact_notification_buttons_dialog_title">Too Many Items</string>
+    <string name="pref_compact_notification_buttons_dialog_error">Select less than %1$d items.</string>
     <string name="pref_lockscreen_background_title">Set Lockscreen Background</string>
-    <string name="pref_lockscreen_background_sum">Set the lockscreen background to the current episode\'s image. As a side effect, this will also show the image in third party apps.</string>
+    <string name="pref_lockscreen_background_sum">Use current episode\'s image in external players and as lockscreen image.</string>
     <string name="pref_enqueue_location_title">Enqueue Location</string>
     <string name="pref_enqueue_location_sum">Add episodes to: %1$s</string>
     <string name="enqueue_location_back">Back</string>
     <string name="enqueue_location_front">Front</string>
     <string name="enqueue_location_after_current">After current episode</string>
-    <string name="pref_smart_mark_as_played_disabled">Disabled</string>
+    <string name="pref_smart_mark_as_played_disabled">Off</string>
     <string name="pref_image_cache_size_title">Image Cache Size</string>
     <string name="pref_image_cache_size_sum">Size of the disk cache for images.</string>
-    <string name="documentation_support">Documentation &amp; Support</string>
-    <string name="visit_user_forum">User forum</string>
-    <string name="bug_report_title">Report bug</string>
+    <string name="documentation_support">Documentation and Support</string>
+    <string name="visit_user_forum">Forum</string>
+    <string name="bug_report_title">Report Bug</string>
```

Comment:
```suggestion
    <string name="bug_report_title">Report Bugs</string>
```
Sends the wrong message?

Reply:
> Sends the wrong message?

Yeah. And also, the system will help you report one bug at a time only. (The stacktrace will overwritten if a second bug happens.)